projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -23,21 +23,15 @@ type BarChartTooltipData = {
 
 interface BarChartProps extends BaseChartProps< SeriesData[] > {}
 
-const BarChart: FC< BarChartProps > = ( {
-	data,
-	margin = { top: 20, right: 20, bottom: 40, left: 40 },
-	withTooltips = false,
-	showLegend = false,
-	legendOrientation = 'horizontal',
-	className,
-	gridVisibility = 'x',
-	width,
-	height = 400,
-} ) => {
+const BarChart: FC< BarChartProps > = props => {
 	const theme = useChartTheme();
 	const { tooltipOpen, tooltipLeft, tooltipTop, tooltipData, hideTooltip, showTooltip } =
 		useTooltip< BarChartTooltipData >();
 
+	const handleMouseLeave = useCallback( () => {
```

Comment:
I don't think this actually does anything because whenever `hideTooltip` changes, `handleMouseLeave` gets updated.

Reply:
Oh yeah you're totally right about that, thanks!